platform/linux-generic/odp_traffic_mngr.c

```diff
@@ -2560,6 +2560,18 @@ int odp_tm_capabilities(odp_tm_capabilities_t capabilities[] ODP_UNUSED,
 	cap_ptr->vlan_marking_supported        = true;
 	cap_ptr->ecn_marking_supported         = true;
 	cap_ptr->drop_prec_marking_supported   = true;
+	cap_ptr->tm_queue_threshold            = true;
```

Comment:
Not really related to the changes at hand, but this function incorrectly marks capabilities parameter with ODP_UNUSED. Now that you are changing this function anyway, you could fix it at the same time if is is ok with you,